sale_cutoff_time_delivery/readme/ROADMAP.rst

```diff
@@ -0,0 +1,2 @@
+* Consider timezones
```

Comment:
This must be fixed as well, although I still don't know how to do it :sweat_smile: